docs/preprocessors/in-general.md

```diff
@@ -0,0 +1,100 @@
+# Using Preprocessors
+
+## Generic setup
+
+If a svelte file contains some language other than `html`, `css` or `javascript`, `svelte-vscode` needs to know how to [preprocess](https://svelte.dev/docs#svelte_preprocess) it. This can be achieved by creating a `svelte.config.js` file at the root of your project which exports a svelte options object (similar to `svelte-loader` and `rollup-plugin-svelte`). It's recommended to use the official [svelte-preprocess](https://github.com/sveltejs/svelte-preprocess) package which can handle many languages.
+
+```js
+// svelte.config.js - NOTE: you cannot use the new "import x from y" and "export const" syntax in here.
+const sveltePreprocess = require('svelte-preprocess');
+
+module.exports = {
+    preprocess: sveltePreprocess(),
+};
+```
+
+It's also necessary to add a `type="text/language-name"` or `lang="language-name"` to your `style` and `script` tags, which defines how that code should be interpreted by the extension.
```

Comment:
Maybe worth mentioning in this line that default languages can also be configured (with details in a section below), so that readers don’t jump to conclusions upon reading “It’s also necessary to add ...”.

Reply:
I totally forgot about syntax highlighting which @jasonlyu123 pointed out. Because of this, I'll keep it as is to not "advertise" this as something this would work for everyone, which it will not (at least syntax highlighting wise).